client/src/pages/SettingsPage/components/AdminSettingsPage/AdminSettings.jsx

```diff
@@ -0,0 +1,136 @@
+import { React, useState }  from 'react';
+import routes from '../../../../configs/routes';
+import { userSelector } from '../../../../redux';
+import { Link } from 'react-router-dom';
+import resset_password_icon from '../../../../assets/images/SettingsPageImages/resset_password_image.svg';
+import username_icon from '../../../../assets//images/SettingsPageImages/username-icon.svg';
+import changes_was_saved_icon from '../../../../assets/images/SettingsPageImages/changes_was_saved.svg';
+import cancel from '../../../../assets/images/SettingsPageImages/cancel.svg';
+import style from './AdminSettings.module.scss';
+import SaveButton from '../../../../shared/Buttons/Save';
```

Comment:
You can import SaveButton from Buttons, like: 
```suggestion
import { Save } from '../../../../shared/Buttons';
```